uint32 id
---
bool is_visible
float32[2] top_left
float32[2] bottom_right
float32 x_center
float32 area